spark-testing/spark_utils.py

```diff
@@ -147,14 +148,26 @@ def submit_job(
 
     if write_conf_to_temp_file:
         args_file = tempfile.NamedTemporaryFile("w")
-        args_file.write(submit_args)
+        args_file.write('dcos spark run {} --submit-args="{}"'.format(verbose_flag, submit_args))
```

Comment:
If you write the entire command to a file, you can just execute the file as a shell script: like
`subprocess.run(args_file.name)`
rather than cat-ing the contents of the files.